src/applications/vaos/tests/appointment-list/components/ConfirmedAppointmentDetailsPage/index.unit.spec.js

```diff
@@ -4,34 +4,20 @@ import moment from 'moment-timezone';
 import MockDate from 'mockdate';
 import { mockFetch } from '@department-of-veterans-affairs/platform-testing/helpers';
 import userEvent from '@testing-library/user-event';
-import sinon from 'sinon';
-import { fireEvent, waitFor } from '@testing-library/react';
-import {
-  APPOINTMENT_STATUS,
-  TYPE_OF_VISIT_ID,
-} from '../../../../utils/constants';
+import { waitFor } from '@testing-library/react';
+import { APPOINTMENT_STATUS } from '../../../../utils/constants';
 import { renderWithStoreAndRouter, getTestDate } from '../../../mocks/setup';
 
 import { AppointmentList } from '../../../../appointment-list';
 import {
   mockAppointmentApi,
-  mockAppointmentUpdateApi,
-  mockClinicsApi,
   mockGetPendingAppointmentsApi,
   mockGetUpcomingAppointmentsApi,
-  mockSingleVAOSAppointmentFetch,
 } from '../../../mocks/helpers';
-import { getVAOSAppointmentMock } from '../../../mocks/mock';
-import {
-  mockFacilitiesFetch,
-  mockFacilityFetch,
-  mockSingleClinicFetch,
-} from '../../../mocks/fetch';
-import { createMockFacility } from '../../../mocks/data';
+import { mockFacilitiesFetch } from '../../../mocks/fetch';
 import MockAppointmentResponse from '../../../e2e/fixtures/MockAppointmentResponse';
-import MockFacilityResponse from '../../../e2e/fixtures/MockFacilityResponse';
-import MockClinicResponse from '../../../e2e/fixtures/MockClinicResponse';
 
+// TODO: Delete this test suite since these test cases are now covered by the various direct schedule appointment layouts.
```

Comment:
Instead of `TODO`s which are easy to lose track of, file an issue and link to it in the comments.

Reply:
This is an old comment that I forgot to delete.